djadmin2/tests/test_views.py

```diff
@@ -1,7 +1,13 @@
-from django.test import TestCase
+from django.core.urlresolvers import reverse
+from django.test import TestCase, override_settings
+from django.utils.encoding import force_text
+from os import path
 
 from .. import views
 
+fixture_dir = path.join(path.abspath(path.dirname(__file__)), 'fixtures')
+fixture_file = path.join(fixture_dir, 'pubtest.txt')
```

Comment:
@andrewsmedina Probably nowhere...

I just forgot to delete them after a copy/paste, it should now be better